pulp_ansible/app/models.py

```diff
@@ -0,0 +1,43 @@
+from logging import getLogger
+
+from django.db import models
+
+from pulpcore.plugin.models import Content, Importer, Publisher
+
+
+log = getLogger(__name__)
+
+
+class AnsibleRole(Content):
+    """
+    A content type respresenting an ansible role
+    """
+
+    TYPE = 'ansible'
+
+    namespace = models.CharField(max_length=64)
+    name = models.CharField(max_length=64)
+    version = models.CharField(max_length=128)
+
+    class Meta:
+        unique_together = (
+            'namespace',
+            'name',
+            'version'
+        )
+
+
+class AnsiblePublisher(Publisher):
```

Comment:
Will this publisher only handle `AnsibleRole` stuff or will it handle other types in the future as well like `TheNextAnsibleThing`? If it's always-and-forever `AnsibleRole`, should this be called `AnsibleRolePublisher`?

Reply:
My assumption (at least for now) was that we'd want to publish all Ansible content in a repo together (not just roles).

Reply:
Great then this is the right name. Ack.